api/python/provisioner/ui/README.md

```diff
@@ -0,0 +1,10 @@
+# Provisioner TUI
+
+This is northbound interface TUI for CORTX components that provides terminal UI
+to configure system.
+
+## start with provisioner TUI
+
+1. command to start provisioner TUI
+
+-  `python3 api/python/provisioner/ui/main.py`
```

Comment:
The command should not be main.py.
Please move this to under the command API and expose it under command:
`provisioner configure_node`
later to be converted to:
`cortx_setup configure_node`

Reply:
facing some issue directly calling TUI from API, will cover this comment in next ticket. EOS-18273